plasmapy/constants/atomic.py

```diff
@@ -0,0 +1,1187 @@
+    <Quantity 238.02891 u>
+    >>> standard_atomic_weight(118) is None
+    UserWarning: No standard atomic weight is available for Og. Returning None.
+    True
+
+    """
+
+    try:
+        isotope = isotope_symbol(argument)
+    except:
+        isotope = ''
+
+    if type(argument) == str and (argument in ['p', 'p+'] or
+       argument.lower() in ['proton', 'alpha']):
+        raise ValueError("Use ion_mass to get masses of protons and alpha "
+                         "particles instead of standard_atomic_weight")
+    elif isotope == 'n':
+        raise ValueError("Use isotope_mass('n') or plasmapy.constants.m_n "
+                         "instead of standard_atomic_weight to get neutron "
+                         "mass")
+    elif '-' in isotope or isotope in ['D', 'T']:
+        raise ValueError("Use isotope_mass to get masses of isotopes")
+
+    atomic_symbol = element_symbol(argument)
+
+    try:
+        atomic_weight = Elements[atomic_symbol]['atomic_mass']
+    except:
+        raise ValueError("No standard atomic weight is available for " +
+                         atomic_symbol)
+
+    return atomic_weight
+
+
+def isotope_mass(argument, mass_numb=None):
+    """Return the mass of an isotope.
+
+    Parameters
+    ----------
+    argument : string or integer
+        A string representing an element or isotope or an integer representing
+        an atomic number
+
+    mass_numb : integer (optional)
+        The mass number of the isotope.
+
+    Returns
+    -------
+    isotope_mass : Quantity
+        The atomic mass of a neutral atom of an isotope.
+
+    Raises
+    ------
+    UserWarning:
+        Redundant (but consistent) isotope information is provided.
+
+    ValueError:
+        Contradictory or insufficient isotope information is provided.
+
+    See also
+    --------
+    standard_atomic_weight : returns atomic weight of an element based on
+        terrestrial abundances of isotopes
+
+    ion_mass : returns the mass of an ion of an element or isotope, accounting
+        for loss of electrons
+
+    Notes
+    -----
+    The masses of rare isotopes may be unavailable.
+
+    Examples
+    --------
+    >>>from astropy import units as u
+    >>>isotope_mass("H-1")
+    <Quantity 1.00782503223 u>
+    >>>isotope_mass("H-1").to(units.kg)
+    <Quantity 1.6735326915759943e-27 kg>
+    >>> isotope_mass("He", 4)
+    <Quantity 4.00260325413 u>
+    >>> isotope_mass(2, 4)
+
+    """
+
+    if argument == 'alpha':
+        raise ValueError("Use ion_mass for mass of an alpha particle")
+    elif argument in ['p', 'proton']:
+        raise ValueError("Use plasmapy.constants.m_p or ion_mass('p') to get "
+                         "proton mass")
+
+    symbol = isotope_symbol(argument, mass_numb)
+    atomic_mass = Isotopes[symbol]['atomic_mass']
+
+    return atomic_mass
+
+
+def ion_mass(argument, Z=1, mass_numb=None):
+    """Returns the mass of an ion.
+
+ by finding the atomic mass of a neutral
```

Comment:
Is the indentation wrong here?  Is part of your travis build a documentation build?